test/integration/controllers/review.spec.js

```diff
@@ -26,7 +26,10 @@ let offerBody = {
   proficiencyLevel: ['Beginner'],
   description: 'TEST 123ASD',
   languages: ['Ukrainian'],
-  category: '63525e23bf163f5ea609ff27'
+  category: {
+    _id: '63525e23bf163f5ea609ff27',
```

Comment:
have you tried to run this test multiple times? pre-selected id may cause errors

Reply:
it works, but change it, now take data from received category